code/index.html

```diff
@@ -3,28 +3,143 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>dogGO</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://use.typekit.net/epf8jwq.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <style></style>
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+    <!--HEADER SECTION-->
+    
+    <div class="navbar-container">
+      <header class="navbar">
+        <div class="hamburger-wrapper">
+          <div class="hamburger">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+        </div>
+        <h2 class="logo">     
+          <img src="./images/logo.png" alt="DogGO logo image">
+        </h2>
+        <div class="menu-container">
+          <nav>
+            <ul class=nav-menu>
+              <li><a href="#">Dog-stinations</a></li>
+              <li><a href="#">Off-Leash Experiences</a></li>
+              <li><a href="#">Booking</a></li>
+              <li><a href="#">A-BARK Us</a></li>
+              <li><a href="#">SIGN IN</a></li>
+            </ul>
+          </nav> 
+        </div> 
+      </header>
+    </div>
+
+    <!-- HERO SECTION -->
+
+    <section class="hero-section">
+      <div class="hero-header">
+        <h1>An Adventure<br>you can both enjoy!</h1>
+      </div>
+
+      <div class="hero-container" id="hero">
+        <video class="video" loop autoplay muted preload>
+          <source src="./videos/Herovideo_1280x720.mp4" type="video/mp4">
+          <source src="./videos/Herovideo_1280x720.webm" type="video/webm">
+        
+          <source class="tablet" src="./videos/Herovideo_720x480.mp4" media="(max-width: 991px)" type="video/mp4">
+          <source class="tablet" src="./videos/Herovideo_720x480.webm" media="(max-width: 991px)" type="video/webm">  
+        </video> 
+        <img class="hero-img" src="./images/Hero_mobileimg.jpg" alt="Man running with dog beside him">      
+      </div>   
+    </section> 
+
+    <!-- SIGN UP FORM -->
+  <div class="the-big-container">
+    <section class=form>
+      <form action="https://httpbin.org/anything" method="post">
+        <h1 class=form-title>Sign Up for More Info!</h1>
+        <ul class="form-info">
+          <li>
+            <label for="name">Name</label>
+            <input type="text" id="name" name="user_name">
+          </li>
```

Comment:
Here you can style the `label`s and `input`s and clean out the `ul` and `li` elements :)